hokusai/commands/check.py

```diff
@@ -53,16 +56,11 @@ def check_err(check_item):
     check_err('git')
     return_code += 1
 
-  if os.environ.get('AWS_ACCESS_KEY_ID') is not None:
-    check_ok('$AWS_ACCESS_KEY_ID')
-  else:
-    check_err('$AWS_ACCESS_KEY_ID')
-    return_code += 1
-
-  if os.environ.get('AWS_SECRET_ACCESS_KEY') is not None:
-    check_ok('$AWS_SECRET_ACCESS_KEY')
-  else:
-    check_err('$AWS_SECRET_ACCESS_KEY')
+  try:
+    boto3.client('sts', region_name=get_region_name()).get_caller_identity()
+    check_ok('Valid AWS credentials')
+  except botoexceptions.ClientError, botoexceptions.NoCredentialsError:
+    check_err('Valid AWS credentials')
```

Comment:
`s/Valid/Invalid`

Reply:
I was a bit confused by the interface here but I think this expands to an error message of `Valid AWS credentials not found`.

Reply:
Oops!  Right you are.